mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -2,7 +2,7 @@
   "tables": [
     {
       "tableName": "job_executions",
-      "fromModuleVersion": "mod-source-record-manager-3.0.5",
+      "fromModuleVersion": "mod-source-record-manager-3.0.6",
```

Comment:
By default it won't be released as a hotfix, please use the 3.1.0 version. If it is crucial for Iris, then we'll change it

Reply:
Changed